new-chapter.md

```diff
@@ -0,0 +1,38 @@
+# PyLadies Request to Start Chapter Form
+
+Please fill in the following form to the best of your knowledge, and keep in mind that PyLadies is managed on a volunteer basis and the approval or feedback on the application might take a while. We will contact you as soon as we can.
+
+## Organizer
+
+- `your name`
+- `what are your pronouns (e.g. she/her, they/them)?`
+- `your contact email`
+- `username on PyLadies slack (Register first at https://slackin.pyladies.com/.)`
+- `Who are your co-organizer(s) please include their name and pronouns? We advise having at least 2 organizers a chapter to start.`
+- `What are your co-organizer(s) emails?`
+- `What are your co-organizer(s) PyLadies Slack user names?`
+
+## Background
+
+- `How did you hear about PyLadies?`
+- `Have you attended PyLadies meetups in other cities? If yes where?`
+- `How do you and your organizer(s) use Python? Organizer(s) of all background levels are welcome! Understanding your relationship to Python is helpful for PyLadies to support your chapter as you get started.`
+
+## Chapter Information
+
+- `City of new chapter. PyLadies recommends a city specific focus to prevent name collision. If you want to start a chapter with a different affiliation please specify the location and why you are choosing this.`
+- `Country of new chapter`
+- `Desired namespace for chapter, this will be for the email and subdomain setup like [sf@pyladies.com] and it should be as close to the chapter name as possible. Remember, in Python, explicit is better than implicit!`
+- `What language(s) will the chapter primarily speak?`
+
+## Programming
+
+- `When do you plan to have your first event?`
+- `What type of event(s) do you plan to run?`
+- `Do you have any specific support requests? For example, does your community speak a language other than English and will you want translation support?`
+- `Do you need mentorship from chapters in cities near by? If so which chapters?`
+
+## Agreement
+
+- `I agree to await for official approval from PyLadies before starting to build resources online.`
+- `I confirm that I read and understood our data privacy policy: PyLadies is committed to ensuring the security and protection of the personal information that we process, and to provide a compliant and consistent approach to data protection. We require the above information solely for internal record keeping and getting in contact regarding matters with respect to chapter management.`
```

Comment:
@terezaif this sounds like a good start! I wonder if we should make it a bit broader though like:

`We require the above information for PyLadies chapter management, internal record keeping and other related functions for managing the PyLadies brand that requires organizers input.`

Reply:
sounds good!